core/nodejsActionBase/app.js

```diff
@@ -17,7 +17,8 @@
 
 var config = {
         'port': 8080,
-        'apiHost': process.env.__OW_API_HOST
+        'apiHost': process.env.__OW_API_HOST,
+        'allowConcurrent': process.env.OW_ALLOW_CONCURRENT
```

Comment:
do you think we'll want to make `__OW_ALLOW_CONCURRENT` part of the activation context at some point? if so, using `__` prefix would be uniform.

Reply:
I agree, but until that is done in OW core code, this seemed less confusing. I think there is some longer discussion about how that is done in OW core (runtime manifest?). I can go ahead and change it here in anticipation though.